### ISSUE TITLE
Can't copy worksheet with multiple tables in some workbooks

### ISSUE THREAD

**Issue by craigbrown** [EPPlusSoftware/EPPlus#961]
I have a workbook with a sheet that makes EPPlus throw an exception when trying to copy it - see attached.

I'm not exactly sure why it's happening. It's something to do with the fact that the sheet has two tables of differing widths. I think it's also something to do with the order/numbering of the `tablePart` elements in the sheet XML.

I did some digging and tracked down the issue to [WorksheetCopyHelper.cs:line 688](https://github.com/EPPlusSoftware/EPPlus/blob/4406647a2b8b5270729c080dcb660dadaa9813a3/src/EPPlus/Core/Worksheet/WorksheetCopyHelper.cs#L688). This line finds a `tablePart` element in the _new_ worksheet by using the ID of the table in the _existing_ worksheet. The second time the code hits that line, the _new_ worksheet has two `tablePart` elements with the same ID - so it picks the wrong one.

That's as far as I got - hope that's of some help!

Here's a unit test as well. Many thanks!

```csharp
[TestMethod]
public void CopyWorksheetBug()
{
    using (var pck = new ExcelPackage("TwoTablesTest.xlsx"))
    {
        // Create a new workbook
        using (var newPck = new ExcelPackage())
        {
            // Copy TestTables to the new workbook
            newPck.Workbook.Worksheets.Add("TestTablesNew", pck.Workbook.Worksheets["TestTables"]);
            var copiedSheet1 = newPck.Workbook.Worksheets["TestTablesNew"];
            Assert.IsNotNull(copiedSheet1);
        }
    }
}
```

[TwoTablesTest.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/12021806/TwoTablesTest.xlsx)

**Comment by OssianEPPlus** [comment]
Hi! Thank you so much for providing a sheet and a test with your original issue!

This is a bug in our copy function where internally in some cases the order of tables get mixed up after copying.
We're working on a fix. Thank you so much for bringing this to our attention!

I will update you as soon as we've resolved the issue

**Comment by JanKallman** [comment]
Fixed i 6.2.7
